### PR TITLE
Update to Muon interface release notes

### LINKED ISSUE #26114: Documentation for new MUON GUI and elemental analysis
Release notes too!

- [ ] new muon gui
- [x] elemental analysis #26242
- [ ] release notes

### PR DESCRIPTION
**Pull request by BassoLorenzo**
Updated the release notes for the Muon interface.

**To test:**

Fixes #26114.
Do not merge until after  #26242.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by Harrietbrown** [comment]
What does deleting ``CreateSampleShape-v1.rst`` has to do with this PR?

**Comment by LolloB** [comment]
> What does deleting `CreateSampleShape-v1.rst` has to do with this PR?

Nothing, I had deleted it as it does not compile on Ubuntu 18.04+ and forgotten to reintroduce it.

File: docs/source/release/v4.1.0/muon.rst [hunk 1]
```diff
@@ -18,6 +18,8 @@ New
 * Frequency Domain Analysis GUI added to workbench.
 * Muon ALC GUI added to workbench.
 * Added phase tab for calculating :ref:`phase tables <algm-CalMuonDetectorPhases>` and :ref:`PhaseQuad <algm-PhaseQuad>` workspaces to Frequency Domain Analysis GUI.
+* Muon Analysis v2 added to MantidPlot and workbench
```
Comment (AnthonyLim23) [inline docs/source/release/v4.1.0/muon.rst]:
```suggestion
* Muon Analysis v2 added to MantidPlot and workbench.
```

File: docs/source/release/v4.1.0/muon.rst [hunk 2]
```diff
@@ -18,6 +18,8 @@ New
...
+* Elemental analysis interface added to workbench
```
Comment (AnthonyLim23) [inline docs/source/release/v4.1.0/muon.rst]:
```suggestion
* Elemental analysis interface added to workbench.
```

**Comment by AnthonyLim23** [comment]
can we have more details about the new muon analysis GUI and a link to its documentation. Please stress how it is designed to work better with multiple runs, be more intuitive and more stable. Add anything else you think is appropriate.

**Comment by Harrietbrown** [comment]
Elemental Analysis itself really needs it's own documentation. I think that is what is implied in the issue.

**Comment by AnthonyLim23** [comment]
@Harrietbrown we have documentation on the elemental analysis GUI in a PR #26242

@NickDraper is this the sort of thing you had in mind?

**Review by NickDraper — CHANGES_REQUESTED** [review]
I'd like to see a screenshot added and a list of the best points about the new GUI, ideally why it is better than the old one, and what it lets people do.  All aimed at our users.

This is the culmination of about 1-5 - 2 man years of work, and this just does not do that credit.

**Comment by AnthonyLim23** [comment]
Could you also add the fit and results table link to the frequency domain docs.

File: docs/source/release/v4.1.0/muon.rst [hunk 3]
```diff
@@ -15,15 +31,34 @@ MuSR Changes
 New
 ###
 
-* Frequency Domain Analysis GUI added to workbench.
-* Muon ALC GUI added to workbench.
-* Added phase tab for calculating :ref:`phase tables <algm-CalMuonDetectorPhases>` and :ref:`PhaseQuad <algm-PhaseQuad>` workspaces to Frequency Domain Analysis GUI.
+* :ref:`Frequency Domain Analysis <Frequency_Domain_Analysis_2-ref>` GUI added to workbench.
```
Comment (Harrietbrown) [inline docs/source/release/v4.1.0/muon.rst]: Workbench is a proper noun and should be capitalized like you would MantidPlot

```suggestion
* :ref:`Frequency Domain Analysis <Frequency_Domain_Analysis_2-ref>` GUI added to Workbench.
```

File: docs/source/release/v4.1.0/muon.rst [hunk 4]
```diff
@@ -15,15 +31,34 @@ MuSR Changes
...
+* :ref:`Muon ALC GUI <MuonALC-ref>` added to workbench.
```
Comment (Harrietbrown) [inline docs/source/release/v4.1.0/muon.rst]:
```suggestion
* :ref:`Muon ALC GUI <MuonALC-ref>` added to Workbench.
```

File: docs/source/release/v4.1.0/muon.rst [hunk 5]
```diff
@@ -15,15 +31,34 @@ MuSR Changes
...
+* Added phase tab for calculating :ref:`phase tables <algm-CalMuonDetectorPhases>` and :ref:`PhaseQuad <algm-PhaseQuad>`
+  workspaces to Frequency Domain Analysis GUI.
+* :ref:`Muon Analysis v2 <MuonAnalysis_2-ref>` added to MantidPlot and workbench.
```
Comment (Harrietbrown) [inline docs/source/release/v4.1.0/muon.rst]:
```suggestion
* :ref:`Muon Analysis v2 <MuonAnalysis_2-ref>` added to MantidPlot and Workbench.
```

File: docs/source/release/v4.1.0/muon.rst [hunk 6]
```diff
@@ -15,15 +31,34 @@ MuSR Changes
...
+* :ref:`Elemental analysis <Muon_Elemental_Analysis-ref>` interface added to workbench.
```
Comment (Harrietbrown) [inline docs/source/release/v4.1.0/muon.rst]:
```suggestion
* :ref:`Elemental analysis <Muon_Elemental_Analysis-ref>` interface added to Workbench.
```

File: docs/source/interfaces/Frequency Domain Analysis.rst [hunk 7]
```diff
@@ -46,6 +46,8 @@ All these tabs can be dragged away from the interface into their own window.
 * :ref:`Grouping Tab <muon_grouping_tab-ref>`
 * :ref:`Phase Table Tab <muon_phase_table_tab-ref>`
 * :ref:`Transform Tab <muon_transform_tab-ref>`
+* :ref:`Fitting Tab <muon_fitting_tab-ref>`
+* :ref:`Results Table Tab <muon_results_tab-ref>`
```
Comment (Harrietbrown) [inline docs/source/interfaces/Frequency Domain Analysis.rst]: On the `muon_fitting_tab.rst` and `muon_results_tab.rst` documents add Frequency domain analysis to the "User By" section.

**Review by Harrietbrown — COMMENTED** [review]
A few notes on capitalization, and make sure that documentation for the new FDA tabs are updated as well to say that they are used by FDA.

File: docs/source/interfaces/muon_results_tab.rst [hunk 8]
```diff
@@ -56,3 +56,4 @@ Used By
 ^^^^^^^
 
 :ref:`Muon Analysis <MuonAnalysis_2-ref>`
```
Comment (Harrietbrown) [inline docs/source/interfaces/muon_results_tab.rst]:
```suggestion
:ref:`Muon Analysis 2 <MuonAnalysis_2-ref>`
```

**Review by Harrietbrown — CHANGES_REQUESTED** [review]

**Review by Harrietbrown — APPROVED** [review]
Very good. Well writen and the improvement section in particular is very positive.

**Review by NickDraper — APPROVED** [review]

**Pull request merged**